config.yaml

```diff
@@ -13,7 +13,63 @@ environment:
     auto_detect: True
 weather:
     station: mongo
+    web_service:
+      name: 'aat-metdata'
+      link: 'http://site.aao.gov.au/met/metdata.dat'
```

Comment:
Link is changed.
I am assuming that the order of the elements in the file are the same as the last one.

Reply:
Format is the same, but the rain & wet flags are just -1, 0 or 1 (no data, not wet, wet) instead of integers 0 to 10 (number of wet readings in the last 10 minutes).